V0.5.2BeBuilder/bebuilder.rdef

```diff
@@ -0,0 +1,70 @@
+
+resource app_signature "application/x-vnd.bebuilder";
+
+resource app_version {
+	major  = 0,
+	middle = 0,
+	minor  = 0,
```

Comment:
Maybe middle and minor could be changed to:
```
	middle = 5,
	minor  = 2,
```

BTW, in  V0.5.2BeBuilder/makefile you replaced
```
RSRCS= bebuilder.rsrc \
		# ./Resources/ViewResource.rsrc \
		./Resources/WindowResource.rsrc
```
by
```
RDEFS= bebuilder.rdef \
		./Resource/ViewResource.rdef \
		./Resources/WindowResource.rdef
```
So I was wondering if the ./Resource/ViewResource.rdef should be commented out, like the ./Resources/ViewResource.rsrc was.
(I hope what I wrote makes sense ;-)